app/(main)/register/page.test.tsx

```diff
@@ -47,7 +47,11 @@ describe('Register', () => {
   beforeEach(() => {
     jest.clearAllMocks();
 
-    render(<Register />);
+    render(
+      <div className="dark">
```

Comment:
Is this required? I thought our default tailwind theme was already dark?

Reply:
Ah, I thought I had to "mock' per se the dark theme in the testing, but I removed the `div` tag and the test is still passing. Thanks for bringing this up!